components/MainLayout/components/MobileNavbar/index.js

```diff
@@ -33,10 +33,10 @@ class MobileNavbar extends Component {
         <div className={scss.navbar__graphic}>
           <Link href="/">
             <a>
-              <img src="/static/mobile/logo/dpla_bws-logo-color-nav-mobile.png" />
+              <img src="/static/mobile/logo/dpla_bws-logo-color-nav-mobile.png" alt="Black Women's Suffrage Logo"/>
             </a>
           </Link>
-          <img src={menuIcon} onClick={this.showMenu} name="menuDisplayed" id={scss.menuIcon} />
+          <img src={menuIcon} onClick={this.showMenu} name="menuDisplayed" id={scss.menuIcon} alt=""/>
```

Comment:
Does this get shown to screenreaders? Or would they only ever see the other navbar?

Reply:
I think this would get read on a screen reader. It popped up as an issue when I used Lighthouse

Reply:
I guess I'm wondering if this needs alt text because it's the only way someone with a screen reader can navigate the site?